cicecore/cicedynB/general/ice_state.F90

```diff
@@ -192,6 +192,8 @@ subroutine alloc_state
       n_trcr_strata = 0
       nt_strata = 0
       trcr_base = c0
+      aicen = c0
+      aicen_init = c0
```

Comment:
For completeness, we should probably add vsnon/vsnon_init and vicen/vicen_init.

Reply:
I can add these initializations.